lib/platform/gitlab/index.ts

```diff
@@ -936,20 +936,27 @@ export async function addAssignees(
 ): Promise<void> {
   logger.debug(`Adding assignees '${assignees.join(', ')}' to #${iid}`);
   try {
-    let assigneeId = await getUserID(assignees[0]);
-    let url = `projects/${config.repository}/merge_requests/${iid}?assignee_id=${assigneeId}`;
-    await gitlabApi.putJson(url);
-    try {
-      if (assignees.length > 1) {
-        url = `projects/${config.repository}/merge_requests/${iid}?assignee_ids[]=${assigneeId}`;
-        for (let i = 1; i < assignees.length; i += 1) {
-          assigneeId = await getUserID(assignees[i]);
-          url += `&assignee_ids[]=${assigneeId}`;
+    if (assignees.length === 1) {
+      logger.debug('Setting single assignee');
+      const assigneeId = await getUserID(assignees[0]);
+      const url = `projects/${config.repository}/merge_requests/${iid}?assignee_id=${assigneeId}`;
+      await gitlabApi.putJson(url);
+    } else {
+      try {
+        logger.debug('Try using GitLab premium API');
```

Comment:
```suggestion
        logger.debug('Try using GitLab Premium API');
```

Should probably capitalize Premium as well, to be consistent with the way GitLab stylizes this offering... 😄